hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -411,20 +435,24 @@ set_false_path -from [get_ports IOC7] -to [get_ports IOR*]
 # I/O drive/load    #
 #####################
 
-# attach load and drivers to IOs to get a more realistic estimate
-set_driving_cell -no_design_rule -lib_cell ${DRIVING_PAD} -pin ${DRIVING_PAD_PIN} [all_inputs]
-set_load [load_of ${LOAD_PAD_LIB}/${LOAD_PAD}/${LOAD_PAD_PIN}] [all_outputs]
+# This is not needed by CDC runs
```

Comment:
long term do you think we need to split the constraints up into the clocking portion and the "other" stuff?
technically all the other things don't really matter to cdc, we only care about the clocks.

Reply:
It probably makes sense to separate the non-timing constraints into a separate file, yes.